packages/forms/src/UIForm/fields/TextArea.js

```diff
@@ -0,0 +1,54 @@
+import React, { PropTypes } from 'react';
+import FieldTemplate from './FieldTemplate';
+
+export default function TextArea(props) {
+	const { id, isValid, errorMessage, onChange, schema, value } = props;
+	const { autoFocus, description, disabled, placeholder, readOnly, title, type } = schema;
+
+	return (
+		<FieldTemplate
+			description={description}
+			errorMessage={errorMessage}
+			id={id}
+			isValid={isValid}
+			label={title}
+			labelAfter
+		>
+			<textarea
+				id={id}
+				autoFocus={autoFocus}
+				className="form-control"
+				disabled={disabled}
+				label={title}
+				placeholder={placeholder}
+				onChange={event => onChange(event, schema, event.target.value)}
+				readOnly={readOnly}
+				type={type}
+				value={value}
+			/>
```

Comment:
The idea is to validate it through the same validation process as other validation